site/public/css/rainbow-customization.css

```
@@ -127,7 +127,7 @@
     flex-direction: column;
 }
 
-i.fa-exclamation-triangle {
+i.rainbow-grades-warning-triangle {
```

Since `rainbow-grades-warning-triangle` uniquely identifies the target items, we don't need to add additional qualifiers by specifying that the selected element has to be an `<i>` tag.  Apply throughout.

```suggestion
.rainbow-grades-warning-triangle {
```